docs/src/main/asciidoc/elasticsearch.adoc

```diff
@@ -191,13 +227,40 @@ public class FruitService {
         }
         return results;
     }
+
+    private static String toNdJsonString(List<JsonObject> objects) {
+        return objects.stream()
+                .map(JsonObject::encode)
+                .collect(Collectors.joining("\n", "", "\n"));
+    }    
 }
 ----
 <1> We inject an Elasticsearch low level `RestClient` into our service.
 <2> We create an Elasticsearch request.
 <3> We use Vert.x `JsonObject` to serialize the object before sending it to Elasticsearch, you can use whatever you want to serialize your objects to JSON.
 <4> We send the request (indexing request here) to Elasticsearch.
-<5> In order to deserialize the object from Elasticsearch, we again use Vert.x `JsonObject`.
+<5> As we `index` collection of objects we should use `index`, `create` or `update` action.
+<6> We use `toNdJsonString(entityList)` call to produce output like below
++
+[source, json]
+----
+{"index", {"_index" : "fruits", "_id", "1"}}
+{"id": "1", "name": "apple", "color": "red"}
+...           ...             ...        ...
+{"create", {"_index" : "fruits", "_id", "N"}}
+{"id": "N", "name": "dragonfruit", "color": "pink"}
+----
```

Comment:
```suggestion
{"id": "N", "name": "dragonfruit", "color": "pink"}

----
```
I think we have to have that "extra" new line at the end as that's what Elasticsearch expects to mark the end of the request body. Without it I think the request is rejected, no? (and the same with the other ndjson example)